src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1494,36 +1494,39 @@
     (fs/delete csr-path)
     (log/debug (i18n/trs "Removed certificate request for {0} at ''{1}''" subject csr-path))))
 
-(schema/defn revoke-existing-cert!
-  "Revoke the subject's certificate. Note this does not destroy the certificate.
-   The certificate will remain in the signed directory despite being revoked."
+(schema/defn revoke-existing-certs!
+  "Revoke the subjects' certificates. Note this does not destroy the certificates.
+   The certificates will remain in the signed directory despite being revoked."
   [{:keys [signeddir cacert cacrl cakey infra-crl-path
            infra-node-serials-path enable-infra-crl]} :- CaSettings
-   subject :- schema/Str]
-  (let [serial  (-> (path-to-cert signeddir subject)
-                    (utils/pem->cert)
-                    (utils/get-serial))
+   subjects :- [schema/Str]]
+  (let [serials (map #(-> (path-to-cert signeddir %)
+                       (utils/pem->cert)
+                       (utils/get-serial))
+                  subjects)
         [our-full-crl & rest-of-full-chain] (utils/pem->crls cacrl)
-        new-full-crl (utils/revoke our-full-crl
+        new-full-crl (utils/revoke-multiple our-full-crl
                                    (utils/pem->private-key cakey)
                                    (.getPublicKey (utils/pem->ca-cert cacert cakey))
-                                   serial)
+                                   serials)
         new-full-chain (cons new-full-crl (vec rest-of-full-chain))]
     (write-crls new-full-chain cacrl)
-    (log/debug (i18n/trs "Revoked {0} certificate with serial {1}" subject serial))
```

Comment:
hmm yeah, would be good to get this back somehow. the options I see are:
* log here, but just log everything at once, e.g. "Revoked certificates <all subjects> with serials <all serials>"
* log at the java level, where we access the certs/serials one at a time

the latter feels more "correct", but more annoying, since we'd need a whole new set of releases

Reply:
Agreed... yeah it would be a set of releases for both branches, dang. Hm.

Reply:
We could keep it fairly similar with:
```clj
 (log/debug (i18n/trs "Revoked {0} certificates(s)." (count serials)))
```
The the latter that molly was saying at the trace level?

Might want to ask CS what level / info would be most helpful?

Reply:
Okay. For now I'll add that debug in here.